js/src/wcdl/section/index.js

```diff
@@ -1,14 +1,31 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
 /**
  * Internal dependencies
  */
 import Card from './card';
 import './index.scss';
 
-const Section = ( props ) => {
-	const { title, description, children } = props;
+/**
+ * Renders a section layout with optional blocks.
+ *
+ * @param {Object} props React props.
+ * @param {JSX.Element} [props.title] Section title at the left side.
+ * @param {JSX.Element} [props.description] Section description under the title.
+ * @param {JSX.Element} [props.children] Section content at the right side.
+ * @param {boolean} [props.disabled] Whether display the whole section in disabled style.
+ */
+const Section = ( { title, description, children, disabled } ) => {
+	const className = classnames(
+		'wcdl-section',
+		disabled ? 'wcdl-section--is-disabled' : false
+	);
```

Comment:
💅  Maybe also consider allowing `Section` consumer to specify their own className?

```suggestion
const Section = ( { title, description, children, disabled, className } ) => {
	const className = classnames(
		'wcdl-section',
		disabled ? 'wcdl-section--is-disabled' : false,
		className
	);
```

Reply:
It would have some eslint errors by committing with the suggestion codes. I added in https://github.com/woocommerce/google-listings-and-ads/pull/1268/commits/b93ee757531a6f339b96d267afa52d82365e65b9

Reply:
Nice! No worry about not using the suggested change code, it is understandable and I don't mind that at all, the more important thing is to communicate the idea. Thanks for the commit! 😃 👍